tests/acceptance/pageObjects/FilesPageElement/SharingDialog/collaboratorsDialog.js

```diff
@@ -185,7 +185,9 @@ module.exports = {
             collaboratorResult[attrName] = false
           }
         }
-        await this.api.elementIdClick(collaboratorElementId)
+        this.moveToElement('@collaboratorAccessDetailsDrop', -9, 0)
```

Comment:
I fear this magic numbers will give us problems in the future

Reply:
Changed it so it clicks on the collaborator avatar to remove the access details from the screen https://github.com/owncloud/web/pull/7315/commits/467700c86677b8d5669d88cd9ba1a20f3a297969